src/components/Search/utils.ts

```diff
@@ -154,11 +154,11 @@ export const getSearchModalStyles = (): SystemStyleObject => ({
 
   ".DocSearch-Footer": {
     "--docsearch-footer-height": "space.10",
-    "--docsearch-logo-color": "colors.bodyLight",
+    "--docsearch-logo-color": "colors.body.light",
```

Comment:
I know this is from a prev implementation but curious about this `colors` variable. Do you know why is there? how is it mapped? shouldn't it be just `body.light`?

Reply:
I'll need to find it in the source code so you have better context, but the mapping from the root level here is specifically for values declared in CSS Variables to be resolved correctly.

Reply:
@pettinarip so for context, it starts in the `getCSS()` method (the root of the theme compiling) at the [isCSSVariableToken check](https://github.com/chakra-ui/chakra-ui/blob/38acfe89c5d1f1edc67bbc44e2edd38980ca3e08/packages/core/styled-system/src/css.ts#L63-L72). And in this check is this [token resolver](https://github.com/chakra-ui/chakra-ui/blob/38acfe89c5d1f1edc67bbc44e2edd38980ca3e08/packages/core/styled-system/src/css.ts#LL19C1-L29C2) to search in a map object, hence the value being dot notation starting at the root of the given collection `colors`.

Reply:
Oh interesting.

So, would it be similar as doing
```suggestion
    "--docsearch-logo-color": "var(--eth-colors-body-light)",
```
?

Reply:
It would be converted to that CSS variable, yes.